### PR TITLE
Please review

### PR DESCRIPTION
**Pull request by sigono-chao3**

### CODE REVIEW

File: file [hunk 1]
```diff
@@ -1,3 +1,4 @@
 I am content 
 line 2
 line c
+line d
```
Comment (sigono-chao3) [inline file]: why this is line d not line e?

File: file [hunk 2]
```diff
@@ -1,3 +1,4 @@
 I am content 
 line 2
```
Comment (sigono-chao3) [inline file]: Why this is not line B

**Review by sigono-chao3 — COMMENTED** [review]
wut?

File: file [hunk 3]
```diff
@@ -1,3 +1,4 @@
 I am content
```
Comment (sigono-chao3) [inline file:1]: Wow

**Review by sigono-chao3 — COMMENTED** [review]
hmmm

File: file (hunk 3 above)
Comment (sigono-chao3) [reply file:1]: hey

**Review by sigono-chao4 — CHANGES_REQUESTED** [review]
Not good

**Review by sigono-chao4 — APPROVED** [review]

**Review by sigono-chao4 — CHANGES_REQUESTED** [review]
Changed my mine

File: file [hunk 4]
```diff
@@ -1,3 +1,5 @@
 I am content 
-line 2
+line b
 line c
+line e
+line f
```
Comment (sigono-chao4) [inline file:5]: f

**Pull request merged**